tests/conftest.py

```diff
@@ -161,7 +162,7 @@ async def app(
             umap_params=get_umap_parameters(None),
             serve_ui=False,
         )
-        manager = await stack.enter_async_context(LifespanManager(app))
+        manager = await stack.enter_async_context(LifespanManager(app, startup_timeout=120))
```

Comment:
had to increase the timeout because the postgresql version of the end-to-end tests have become substantially flakier

Reply:
maybe move out as a constant so you can document